src/test/java/de/tum/in/www1/artemis/service/DataExportCreationServiceTest.java

```diff
@@ -229,11 +230,14 @@ private void assertCommunicationDataCsvFile(Path courseDirPath) {
 
     /**
      * Asserts the content of the science events CSV file.
-     * Allows for a 500ns difference between the timestamps due to the reimport from the csv export.
-     * Might cause the test to be flaky if multiple events are created overlapping and matched wrongly.
+     * Allows for a 500ns difference between the timestamps due to the reimport from
+     * the csv export.
+     * Might cause the test to be flaky if multiple events are created overlapping
+     * and matched wrongly.
      *
      * @param extractedZipDirPath The path to the extracted zip directory
-     * @param events              The set of science events to compare with the content of the CSV file
+     * @param events              The set of science events to compare with the
+     *                                content of the CSV file
      */
```

Comment:
I'm not sure, but I feel like it would look cleaner if it would stay in the same line. In these cases I think the line isn't too full